doc/source/building/linux.rst

```diff
@@ -33,11 +33,10 @@ SciPy depend on
 
 * Python header files (typically a package named ``python-dev`` or ``python-devel``)
 
-* Unless you are building from released source packages, the `Cython
-  <https://cython.org/>`__ compiler is necessary (typically in a
-  package named ``cython``). For building recent SciPy, it is possible
-  that you need Cython in a newer version than is available in your
-  distribution.
+* The `Cython <https://cython.org/>`__ and `Pythran <https://pythran.readthedocs.io>`__
+  ahead-of-time compilers are necessary (typically packaged as ``cython``
+  and ``pythran``.  For building recent SciPy, it is possible that you need
```

Comment:
I think I'll just merge at this point--there's a missing parenthesis that can be added here later